src/main/java/core/basesyntax/exception/RegistationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class RegistationException extends Exception {
```

Comment:
```suggestion
public class RegistationException extends RuntimeException {
```
and please remove all ```throws RegistationException``` you added to method headers